lib/data/AbstractData.php

```diff
@@ -10,12 +10,14 @@
  * @version   0.22
  */
 
+namespace PrivateBin\data;
```

Comment:
I will fix those issues when adopting PSR-2 so namespaces will be consistent with case-sensitive filenames.

(funfact is that AFAIR namespace doesn't actually need to start with a capital - like in the case of `yii`, but I will fix that, of course)